plugins/org.jboss.tools.openshift.client/build.properties

```diff
@@ -2,6 +2,15 @@ source.. = src/
 src.includes = *
 src.excludes = src/
 output.. = target/classes
-bin.includes = .,\
-               META-INF/,\
-               lib/*
+bin.includes = META-INF/,\
+               .,\
+               plugin.xml,\
+               plugin.properties,\
+               about.html,\
+               about.ini,\
+               about.mappings,\
+               about.properties,\
+               jboss_about.png,\
+               .options,\
```

Comment:
It this required. I though it was there for our internal testing

Reply:
I agree with jeff about the added files. the only ones required are plugin.properties and plugin.xml.

Reply:
.options is the file that's for the tracing (see https://wiki.eclipse.org/FAQ_How_do_I_use_the_platform_debug_tracing_facility "Turning on debug tracing"). All commits since a long time included this.

Reply:
ok, removing them , trusting your expertise. I'm no big expert here.